src/nidmm/examples/nidmm_fetch_waveform.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/python
+
+import argparse
+import nidmm
+import sys
+import time
+
+parser = argparse.ArgumentParser(description='Performs a waveform acquisition using the NI-DMM API.', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+parser.add_argument('-n', '--name', default='PXI1Slot2', help='Resource name of a National Instruments Digital Multimeter.')
+parser.add_argument('-f', '--function', default='WAVEFORM_VOLTAGE', choices=nidmm.Function.__members__.keys(), type=str.upper, help='Measurement function.')
+parser.add_argument('-r', '--range', default=10, type=float, help='Measurement range.')
+parser.add_argument('-d', '--digits', default=6.5, type=float, help='Digits of resolution for the measurement.')
+parser.add_argument('-s', '--samples', default=10, type=int, help='The number of measurements the DMM makes.')
```

Comment:
don't call this measurements, in waveform mode these are samples

"Number of waveform samples to acquire"

Reply:
Fixed